### DISCUSSION TITLE
What does the red error box say? Why is it clipped?

### DISCUSSION THREAD

**Discussion by HubKing** [nukeop/nuclear#1399] [Q&A]
When I double-click a search result first time, it says some module is missing in a red box, but the message is clipped and there seems to be no way to expand or open it in a bigger window, so I do not know what is missing. (Video at 00:18 )

If I double-click multiple times, eventually the box becomes green and it plays the song. Is this a bug? I installed it using Flatpak. yt-dlp is installed separately. Arch-based.

[bug.webm](https://user-images.githubusercontent.com/10900489/218295738-33134324-9428-4f22-a37f-604a8a3b13ea.webm)

**Comment by nukeop** [comment]
It displays whatever error is returned from the streaming source. If it's too long it's something I haven't seen yet and couldn't plan for. There are many reasons streaming can fail, like geoblocking, problems with your network, etc.

**Reply by HubKing** [reply]
Isn't a way to see the full message otherwise, like some sort of log page?

**Comment by Kiimby** [comment]
I'm having the same problem 
![image](https://user-images.githubusercontent.com/50971210/227740449-455b8e12-94b1-48cb-8101-03f49bd866a6.png)

**Comment by nukeop** [comment]
Logs can be seen in the terminal if you run it from there.

**Reply by Kiimby** [reply]
I'm a windows user

**Reply by HubKing** [reply]
Command Prompt is the same thing.

But I am afraid  that I have to say that it is a bad user interface that the user has to launch the app from command line just to see the error message. I am a bit of a programmer myself, and I do not understand the rationale of "I have never seen a long error message myself, so I won't handle long error messages".

**Reply by nukeop** [reply]
There is none, I just didn't know how to handle it better. The queue item is small so displaying errors there is hard, and it should have better CSS.
